javatests/build/bazel/tests/integration/WorkspaceDriverIntegrationTest.java

```diff
@@ -25,13 +25,41 @@ public void testWorkspaceWithBazelRcFile() throws Exception {
 
         driver.scratchFile(".bazelrc", "test --javacopt=\"-InvalidOpt\"");
 
-        Command cmd = driver.bazel(Optional.of(Paths.get(".bazelrc")), "test", "//:TestMe");
+        Command cmd = driver.bazelCommand()
+                .withBazelrcFile(Paths.get(".bazelrc"))
+                .withArguments("test", "//:TestMe").build();
+
         int returnCode = cmd.run();
 
         assertNotEquals("bazel test return code", 0, returnCode);
         assertTrue("stderr contains InvalidOpt failure", cmd.getErrorLines().stream().anyMatch(x -> x.contains("-InvalidOpt")));
     }
 
+    @Test
+    public void testRunningWithEnvironment() throws Exception {
+        String testName = "test_me";
+        String key = "SOME_KEY";
+        String val = "some_value";
+        driver.scratchFile("BUILD.bazel", shTest(testName));
+        driver.scratchExecutableFile(testName+".sh", shellTestingEnvironmentVariable(key,val));
+        Command cmd = driver.bazelCommand()
+                //.withEnvironmentVariable(key,val)
```

Comment:
I don’t understand. You don’t need the withEnvironmentVariable?

Reply:
this was mistakingly pushed. I was only checking that the test indeed fail if commenting out this line. pardon